.github/workflows/build-release.yaml

```diff
@@ -310,11 +310,110 @@ jobs:
             ${{ steps.generate_package.outputs.package_path }}
             ${{ steps.generate_package.outputs.debug_path }}
 
+  build_mac:
+    strategy:
+      matrix:
+        configuration: [FastDebug, Release]
+        compiler: [clang]
+        arch: [x86_64]
+    name: Mac
+    needs: create_release
+    runs-on: macos-latest
+    steps:
+      - name: Cache Qt
+        id: cache-qt-mac
+        uses: actions/cache@v1
+        with:
+          path: ${{ github.workspace }}/../Qt
+          key: ${{ runner.os }}-QtCache-${{ env.QT_VERSION }}
+      - name: Install Qt
+        uses: jurplel/install-qt-action@v2
+        with:
+          version: ${{ env.QT_VERSION }}
+          dir: ${{ github.workspace }}/..
+          cached: ${{ steps.cache-qt-mac.outputs.cache-hit }}
+          setup-python: 'false'
+          aqtversion: ==1.1.3
+
+      - uses: actions/checkout@v1
+        name: Checkout
+        with:
+          submodules: true
+          fetch-depth: 0
+      - name: Set up test version
```

Comment:
Release workflows do not use `version_override.cmake`, using the strings in `version.cmake` as is.  The release procedure updates the version strings before running the workflow, although this could be a candidate for automating from Github instead of needing to make a commit with the version changes.

Reply:
Removed